docs/sources/explore/trace-integration.md

```diff
@@ -117,7 +117,14 @@ Click the document icon to open a split view in Explore with the configured data
 This feature is currently in beta and behind the `traceToMetrics` feature toggle.
 {{% /admonition %}}
 
-You can navigate from a span in a trace view directly to metrics relevant for that span. This feature is available for Tempo, Jaeger, and Zipkin data sources. Refer to their [relevant documentation](/docs/grafana/latest/datasources/tempo/#trace-to-metrics) for configuration instructions.
+You can navigate from a span in a trace view directly to metrics relevant for that span. This feature is available for Tempo, Jaeger, and Zipkin data sources. Refer to their [relevant documentation](/docs/grafana/latest/datasources/tempo/configure-tempo-data-source#trace-to-metrics) for configuration instructions.
+
+### Trace to profiles
+
```

Comment:
This section would also appear under https://grafana.com/docs/grafana-cloud/send-data/traces/#linking-traces-and-metrics 

Once this PR is reviewed and approved, I'll open another PR for the Cloud docs.

Reply:
we should also consider adding this somewhere in the pyroscope oss docs. Right now it doesn't fit anywhere nicely, but maybe we could have a page that explains profiling + other signals

Reply:
Doc issue: https://github.com/grafana/pyroscope/issues/2662